src/misc/TranslationKey.js

```diff
@@ -473,8 +473,8 @@ export type TranslationKeyType = "about_label"
 	| "giftCardRedeemed_msg"
```

Comment:
You removed this translation in code but not in Phrase and it got pulled in again

Reply:
Done.